integritybackend/actions.py

```diff
@@ -24,20 +25,21 @@
 class Actions:
     """Actions for processing assets.
 
-    All actions operate on:
-        * an "asset", represented by its full path on the local filesystem
-        * some metadata
+    All actions operate on a ZIP file containing three files:
 
-    In an ideal future, all actions would be refactored to accept the exact same
-    inputs: an asset path and a generalized metadata container or configuration
-    object.
+    1. sha256(asset).ext: the asset file with `ext` matching one of the collection's `asset_extensions`
+    2. sha256(asset)-meta-content.json: the metadata associated with the asset file
+    3. sha256(asset)-meta-recorder.json: the metadata associated with the recorder of the asset
     """
 
-    def archive(self, asset_fullpath: str, org_config: dict, collection_id: str):
+    def archive(self, zip_path: str, org_config: dict, collection_id: str):
+        return
+
+    def archive2(self, zip_path: str, org_config: dict, collection_id: str):
```

Comment:
Flagged off `archive` in this PR so it doesn't keep OTS-ing. Will be reverted in subsequent PR.